src/lib/php/Db/DbManager.php

```diff
@@ -402,4 +402,24 @@ public function existsColumn($tableName, $columnName)
     }
     return $this->existsTable($tableName) && $this->dbDriver->existsColumn($tableName, $columnName);
   }
+
+  /**
+   * @param string $sqlStatement
+   * @param array $params
+   * @param string $statementName (optional)
+   * @return null
+   */
+  public function deleteSingleRow($sqlStatement, $params = array(), $statementName = "")
```

Comment:
Function is similar to `getSingleRow()` just without a `return` statement https://github.com/fossology/fossology/blob/fcf1de2f4beb714bd67131cbeeba2604f3ee1fb4/src/lib/php/Db/DbManager.php#L153

Not sure calling it `deleteSingleRow()` and working similar to `getSingleRow()` gives any benefit.